src/Engines/TestWare.Engines.Selenium/Factory/FirefoxDriver.cs

```diff
@@ -0,0 +1,38 @@
+﻿using OpenQA.Selenium.Firefox;
+
+namespace TestWare.Engines.Selenium.Factory;
+
+public class FirefoxDriver : OpenQA.Selenium.Firefox.FirefoxDriver, IBrowserDriver
+{
+    public FirefoxDriver()

```

Comment:
Default constructor doesn't need to be specified
https://www.geeksforgeeks.org/c-sharp-default-constructor/

Reply:
deleted